core/include/keep.h

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
It doesn't really matter, but why was this file renamed? tee_ta_static.h should have been removed and keep.h should have been newly created.